spring-boot-jib/.gitignore

```diff
@@ -0,0 +1,13 @@
+*.class
```

Comment:
This seems unnecessary. I realize several modules have this, but we have a .gitignore at the top-level of the project. So, unless we are adding things here that are unique to jib (e.g. that we don't want to add to the main .gitignore), let's remove this file.

Reply:
Yes, nothing special to Jib, will remove it

Reply:
Removed the file